gcapi/client.py

```diff
@@ -577,20 +582,31 @@ def upload_cases(
         archive: str = None,
         reader_study: str = None,
         answer: str = None,
+        archive_item: str = None,
+        interface: str = None,
     ):
         """
-        Uploads a set of files to an archive or reader study.
+        Uploads a set of files to an archive, archive item or reader study.
         A new upload session will be created on grand challenge to import and
-        standardise your files. This function will return this new upload
+        standardise your file(s). This function will return this new upload
         session object, that you can query for the import status. If this
-        import is successful, the new images will then be added to the selected
-        archive or reader study.
-        You will need to provide the slugs of the objects to pass the images
-        along to. You can find this in the url of the object that you want
+        import is successful, the new image(s) will then be added to the selected
+        archive, archive item or reader study.
+        You will need to provide the slugs of the archive or reader study or the
+        pk of the archive item to pass the images along to.
+        You can find the slug in the url of the object that you want
         to use. For instance, if you want to use the archive at
             https://grand-challenge.org/archives/corads-ai/
         the slug for this is "corads-ai", so you would call this function with
             upload_cases(files=[...], archive="corads-ai")
+        For archive uploads, you can additionally specify the interface slug
+        for the to be created archive items. For archive item uploads, the
+        interface is mandatory. If you define an already existing interface,
+        the old file associated with that interface will be replaced by the new file.
+        You can find a list of interfaces here:
+        https://grand-challenge.org/algorithms/interfaces/
+        The interface slug corresponds to the lowercase hyphenated title of the
+        interface, e.g. generic-medical-image for Generic Medical Image.
```

Comment:
We should add the slug to the table to help users as this is not always the case.